README.md

```diff
@@ -1 +1,55 @@
-# irods_client_http_python
+# iRODS HTTP API Python Wrapper
+
+This is a Python wrapper for the [iRODS HTTP API](https://github.com/irods/irods_client_http_api). 
+
+Documentation for the endpoint operations can be found [here](https://github.com/irods/irods_client_http_api/blob/main/API.md).
+
+## Setup
+**NOTICE:** This project is not yet available through pip. To use, clone the repository into the desired location.
+```
+git clone https://github.com/irods/irods_client_http_python.git
+```
+## Usage
+To use the wrapper, follow the steps listed below.
+
+Import the package.
+```
```

Comment:
For python code snippets, append `py` or `python` to the opening triple backticks.

That'll apply syntax highlighting to the code snippet.